index.js

```diff
@@ -0,0 +1,4 @@
+const title = document.getElementById("title")
+title.addEventListener("click", function () {
+    alert("You clicked the title!!!!!!!!!!!!!!!")
```

Comment:
That's a lot of exclamation points! A bit aggressive. Maybe just three exclamation points would get the point across.

Reply:
Thanks for your comments. I've updated the code accordingly. Please review again.